cli/golem-templates/templates/ts/ts-app-component-websearch-summary-example/components-ts/component-name/src/main.ts

```diff
@@ -12,7 +12,7 @@ import {
   send as sendToLLM
 } from 'golem:llm/llm@1.0.0'
 import { startSearch } from 'golem:web-search/web-search@1.0.0';
-import { env } from 'node:process';
+import { env } from 'process';
```

Comment:
This is definitely not good. It's called `node:process` and there is even a test for it in wasm-rquickjs

Reply:
https://github.com/golemcloud/wasm-rquickjs/blob/main/tests/runtime.rs#L928